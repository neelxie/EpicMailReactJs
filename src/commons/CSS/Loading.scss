

.loading {
  left: 47%;
  position: absolute;
  top: 55%;

  .spinner-border {
    width: 5rem;
    height: 5rem;
    border-color:red;
    vertical-align: text-bottom;
    border-right-color: transparent;
}
}
