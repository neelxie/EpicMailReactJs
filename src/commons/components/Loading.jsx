/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import '../CSS/Loading.scss';

function Loading() {
  return (

    <div className="d-flex justify-content-center loading">
      <div className="spinner-border" role="status">
        <span className="sr-only">Loading...</span>
      </div>
    </div>
  );
}

export default Loading;
