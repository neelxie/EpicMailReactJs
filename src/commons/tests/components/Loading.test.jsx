/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loading from '../../components/Loading';

// test
describe('<Loading> Component', () => {
  it('should match the snapshot', () => {
    const component = shallow(<Loading />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
