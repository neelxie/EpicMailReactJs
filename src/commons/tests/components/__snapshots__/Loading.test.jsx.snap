// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading> Component should match the snapshot 1`] = `
<div
  className="d-flex justify-content-center loading"
>
  <div
    className="spinner-border"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
